DESCRIPTION

```diff
@@ -34,8 +36,10 @@ Imports:
     odbc,
     pool,
     REDCapR,
-    RMySQL
+    RMySQL,
+    xml2
```

Comment:
Was this dependency added by mistake?

Reply:
No - {RMySQL} is the {DBI} driver used when connecting to a MySQL database.

{xml2} is dependency for importing data from Fingertips or DHIS2. It was required for some examples to pass the check.